docs/components/datetime-picker.md

```diff
@@ -276,7 +338,10 @@ A `Number` which sets the custom interval for the Minutes select dropdown.
 
 ### mode
 
-Required `String` prop which accepts the following values: `date`, `time`, `dateTime` and `relative`. The `relative` value is meant to denote an datetime picker instance which only contains relative time frames.
+Required `String` prop which accepts the following values: `date`, `time`, `dateTime`, `relative`. 
+- `relativeOnly`: denotes a datetime picker instance which only contains relative time frames
+- `relativeDate`: instance with relative time frames, and a calendar that supports date selection
+- `relativeDateTime`: instance with relative time frames, and a calendar that supports date and time selection
```

Comment:
I don't understand these docs. The sentence above this lists 4 values that are valid, but then this list shows 3 additional values...?

Reply:
Sorry, I cleaned this up to be a list of all the allowed values. Less confusing now.